demos/ASP.Net MVC/Divergent.ITOps.ViewModelComposition/Divergent.ITOps.ViewModelComposition.csproj

```diff
@@ -109,6 +88,8 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="RequestContext.cs" />
     <Compile Include="StringExtensions.cs" />
+    <Compile Include="Subscription %28generic%29.cs" />
```

Comment:
Thank you,  bit no, once we'll have the problem of a second file I'll be happy to change the convention. AFAIK the CRL name is when the source is reverse engineered, but I might be wrong.